packages/react-components/src/components/Toast.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+// TODO: remove and use the Icon wrapper with correct icon after migration
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+
+import { ToastIcon } from './ToastIcon';
+
+const baseClass = 'lc-toast';
+
+export enum Variants {
+  Success = 'success',
+  Warning = 'warning',
+  Error = 'error',
+  Info = 'info',
+  Notification = 'notification',
+}
+
+interface IToastActionProps {
+  handler: () => void;
+  label: string;
+  closeOnClick?: boolean;
+}
+
+export interface IToastProps {
+  action: IToastActionProps;
+  className?: string;
+  id: string;
+  removable: boolean;
+  variant: Variants;
+  onClose: () => void;
+}
+
+export const Toast: React.FC<IToastProps> = ({
+  action,
+  className,
+  id,
+  children,
+  removable,
+  variant,
+  onClose,
+}) => {
+  const mergedClassNames = cx(
+    baseClass,
+    {
+      [`${baseClass}--${variant}`]: variant,
+    },
+    className
+  );
+
+  const onActionClick = (action: IToastActionProps) => {
+    if (action && action.closeOnClick && onClose) {
+      action.handler;
+      return onClose;
+    }
+
+    return action.handler;
+  };
+
+  return (
+    <div className={mergedClassNames} id={id}>
+      <div className={`${baseClass}__icon`}>
+        <ToastIcon variant={variant} />
+      </div>
+      <div className={`${baseClass}__content`}>{children}</div>
+      {(action || removable) && (
+        <div className={`${baseClass}__actions`}>
+          {action && action.label && action.handler && (
```

Comment:
Do we need to check it if we've defined type for `IToastActionProps` ?

Reply:
[done](https://github.com/livechat/design-system/pull/262/files#diff-a656d3f3036b52add490eb747e708ce34fbc6a1478c86369b71e883eea03dda8R66)